src/App.tsx

```diff
@@ -59,37 +80,73 @@ const useStyles = makeStyles((theme) => ({
     ...theme.mixins.toolbar,
     justifyContent: 'flex-start',
   },
-  content: {
-    flexGrow: 1,
-    padding: theme.spacing(3),
-    transition: theme.transitions.create('margin', {
-      easing: theme.transitions.easing.sharp,
-      duration: theme.transitions.duration.leavingScreen,
-    }),
-    marginRight: -drawerWidth,
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
   },
-  contentShift: {
-    transition: theme.transitions.create('margin', {
-      easing: theme.transitions.easing.easeOut,
-      duration: theme.transitions.duration.enteringScreen,
-    }),
-    marginRight: 0,
+  selectEmpty: {
+    marginTop: theme.spacing(2),
   },
 }));
 
 const App = () => {
     const classes = useStyles();
     const theme = useTheme();
+
     const [open, setOpen] = React.useState(false);
 
+    //Number Input Fields
+    const [heightFt, setHeightFt] = React.useState(0);
+    const [heightIn, setHeightIn] = React.useState(0);
+    const [weight, setWeight] = React.useState(0);
+    const [armspan, setArmspan] = React.useState(0);
+
+    //Color
+    const [color, setColor] = React.useState('');
+
     const handleDrawerOpen = () => {
         setOpen(true);
-        console.log(true);
     };
 
     const handleDrawerClose = () => {
         setOpen(false);
-        console.log(false);
+    };
+
+    const handleOnChange = (event: { target: { value: any; }; }, label: string) => {
+      console.log(event.target.value);
+      console.log(label);
+
+      if(label == "hf"){
+        setHeightFt(event.target.value);
+      }
+      else if(label == "hi"){
+        setHeightIn(event.target.value);
+      }
+      else if(label == "w"){
+        setWeight(event.target.value);
+      }
+      else if(label == "a"){
+        setArmspan(event.target.value);
+      }
+
+      // console.log("hf:" + heightFt + " hi:" + heightIn + " w:" + weight + " a:" + armspan);
+    };
+
+    const handleSubmit = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+      //TODO: send data somewhere and close nav?
+      //TODO: add image check here
```

Comment:
Its rather nit-picky, but can since you have to submit some changes already, can you add a space between where the comment variable starts and where the note starts. It adds some readibility and is common coding practice that I want to implement across languages